src/GitObjectDb/Transformations/PropertyTransformation.MemberVisitor.cs

```diff
@@ -0,0 +1,62 @@
+using GitObjectDb.Attributes;
+using GitObjectDb.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Reflection;
+
+namespace GitObjectDb.Transformations
+{
+    internal partial class PropertyTransformation
+    {
+        [ExcludeFromGuardForNull]
+        internal class MemberVisitor : ExpressionVisitor
+        {
+            public IDictionary<string, object> Values { get; } = new SortedDictionary<string, object>(StringComparer.OrdinalIgnoreCase);
+
+            internal static PropertyInfo ExtractProperty(Expression node)
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this method to not shadow the outer class' member with the same name. ([csharpsquid:S3218](https://sonarcloud.io/organizations/frblondin-github/rules#rule_key=csharpsquid%3AS3218))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=GitObjectDb&pullRequest=40&issues=AWgkNyho1_eUtAp8Im7C&open=AWgkNyho1_eUtAp8Im7C)

[ISSUE_KEY]: AWgkNyho1_eUtAp8Im7C